App.tsx

```diff
@@ -1,21 +1,49 @@
-/**
- * Sample React Native App
- * https://github.com/facebook/react-native
- *
- * @format
- */
-
 import React from 'react';
-import {Text, View} from 'react-native';
-import {SafeAreaView} from 'react-native-safe-area-context';
+import styled from 'styled-components/native';
+import {Provider} from 'react-redux';
+import {store} from './src/app/store';
+import UserIcon from 'react-native-vector-icons/EvilIcons';
+import {CryptoContainer} from './src/components';
 
 function App(): JSX.Element {
```

Comment:
use arrow function and implicit return, if you don't have nothing before the return: 

const App = () => (
<...
)